src/app/backend-api.service.ts

```diff
@@ -470,6 +515,7 @@ export class BackendApiService {
   static GET_PROFILES_ORDER_BY_INFLUENCER_COIN_PRICE = "influencer_coin_price";
   static BUY_CREATOR_COIN_OPERATION_TYPE = "buy";
   static SELL_CREATOR_COIN_OPERATION_TYPE = "sell";
+  static DIAMOND_APP_PUBLIC_KEY = "BC1YLgTKfwSeHuNWtuqQmwduJM2QZ7ZQ9C7HFuLpyXuunUN7zTEr5WL";
```

Comment:
We should decide which key should be stored here

Reply:
The key you are using here looks correct to me